syft/frameworks/torch/tensors/interpreters/precision.py

```diff
@@ -38,10 +40,30 @@ def __init__(
         """
         super().__init__(id=id, owner=owner, tags=tags, description=description)
 
-        self.field = field
         self.base = base
         self.precision_fractional = precision_fractional
         self.kappa = kappa
+        if dtype == "long":
```

Comment:
In the future, (long after this PR is merged), if we wanted to represent something bigger than "long" - how would we do so?

(Hint: please mention something about a separate BigIntTensor abstraction tensor type  - not extending FixedPrecisionTensor with the ability to represent even larger precisions.)

Reply:
Please write a github issue for future work here before merging

Reply:
Okay, I am on it.